src/app/(nowrap_layout)/(session)/profile/page.tsx

```diff
@@ -0,0 +1,735 @@
+"use client";
+
+import { Icon } from '@iconify/react';
+import { capitalCase } from 'change-case';
+import {  useState } from 'react';
+import heartFill from '@iconify/icons-eva/heart-fill';
+import peopleFill from '@iconify/icons-eva/people-fill';
+import roundPermMedia from '@iconify/icons-ic/round-perm-media';
+import roundAccountBox from '@iconify/icons-ic/round-account-box';
+// material
+import { styled } from '@mui/material';
+import { Tab, Box, Card, Tabs, Container } from '@mui/material';
+// redux
+// routes
+// hooks
+// components
+import {
+  Profile,
+  ProfileCover,
+  ProfileFriends,
+  ProfileGallery,
+  ProfileFollowers
+} from '../../../../modules/share/components/_dashboard/user/profile';
+import { UserPost } from '@share/type';
+
+// ----------------------------------------------------------------------
+
+const TabsWrapperStyle = styled('div')(({ theme }) => ({
+  zIndex: 9,
+  bottom: 0,
+  width: '100%',
+  display: 'flex',
+  position: 'absolute',
+  backgroundColor: theme.palette.background.paper,
+  [theme.breakpoints.up('sm')]: {
+    justifyContent: 'center'
+  },
+  [theme.breakpoints.up('md')]: {
+    justifyContent: 'flex-end',
+    paddingRight: theme.spacing(3)
+  }
+}));
+
+// ----------------------------------------------------------------------
+
+export default function UserProfile() {
+  const [currentTab, setCurrentTab] = useState('profile');
+  const [findFriends, setFindFriends] = useState('');
+  const myProfile = {
```

Comment:
mock data should be in seperate file, you could create a folder `__mockData__` in the same module.